src/resolvers/projectPowerResolver.test.ts

```diff
@@ -112,4 +117,180 @@ function projectPowersTestCases() {
     assert.isOk(result);
     assert.equal(result.data.data.powerAmountRank, 4);
   });
+
+  it('must be round independent', async () => {
+    const user1 = await saveUserDirectlyToDb(generateRandomEtheriumAddress());
+
+    const project1 = await saveProjectDirectlyToDb(createProjectData());
+    const project2 = await saveProjectDirectlyToDb(createProjectData());
+    const project3 = await saveProjectDirectlyToDb(createProjectData());
+
+    const roundNumber = project3.id * 10;
+
+    await Promise.all(
+      [
+        [user1, project1, 10], // 1000
+        [user1, project2, 20], // 2000
+        [user1, project3, 30], // 3000
+      ].map(item => {
+        const [user, project, percentage] = item as [User, Project, number];
+        return insertSinglePowerBoosting({
+          user,
+          project,
+          percentage,
+        });
+      }),
+    );
+
+    await takePowerBoostingSnapshot();
+    const incompleteSnapshots = await findInCompletePowerSnapShots();
+    const snapshot = incompleteSnapshots[0];
+
+    snapshot.blockNumber = 1;
+    snapshot.roundNumber = roundNumber - 1;
+    await snapshot.save();
+
+    await insertSinglePowerBalanceSnapshot({
+      userId: user1.id,
+      powerSnapshotId: snapshot.id,
+      balance: 10000,
+    });
+
+    await setPowerRound(roundNumber - 1);
+    await refreshProjectFuturePowerView();
+
+    let result = await axios.post(graphqlUrl, {
+      query: gePowerAmountRankQuery,
```

Comment:
Just noticed the Query is named wrongly it should be 'getPowerAmountRankQuery'
Ill push the renaming.